rerun_py/build.rs

```diff
@@ -3,4 +3,23 @@ fn main() {
     pyo3_build_config::add_extension_module_link_args();
 
     re_build_tools::export_build_info_vars_for_crate("rerun_py");
+
+    // Fail if bin/rerun is missing and we haven't specified it's ok.
+    if re_build_tools::get_and_track_env_var("RERUN_BUILDING_WHEEL").unwrap_or_default() == "1"
```

Comment:
```suggestion
    if re_build_tools::is_tracked_env_var_set("RERUN_BUILDING_WHEEL")
```

Reply:
Perhaps a bit of a confusing name, should probably be called `is_tracked_env_var_true`